core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -149,6 +150,12 @@ private[ui] class ExecutorTable(stageId: Int, stageAttemptId: Int, parent: Stage
                 {Utils.bytesToString(v.diskBytesSpilled)}
               </td>
             }}
+            <td>
+              {val logs = parent.executorsListener.executorToLogUrls.getOrElse(k, Map.empty)
+               logs.map {
+                case (logName, logUrl) => <div><a href={logUrl}>{logName}</a></div>
+              }}
```

Comment:
That's probably what I would do, there a lot of styling up to the coder like this case where scalastyle doesn't complain